content/docs/command-reference/exp/save.md

```diff
@@ -14,33 +20,44 @@ usage: dvc exp save [-h] [-q | -v] [-f]
 
 ## Description
 
-Provides a way to save the current status of your <abbr>project</abbr> as an
-<abbr>experiment</abbr> without polluting it with unnecessary commits, branches,
-directories, etc.
+Provides a way to save a snapshot of your <abbr>project</abbr> as an
+<abbr>experiment</abbr>. This lets you quickly start tracking, [comparing],
+[plotting], and [sharing] experiments after making any project changes (e.g.
+retrain an ML model).
 
-Only files tracked by either Git or DVC are saved to the experiment. Use the
-`--include-untracked` (`-I`) option to explicitly include any untracked files in
-the experiment.
+[comparing]: /doc/user-guide/experiment-management/comparing-experiments
+[plotting]: /doc/user-guide/experiment-management/visualizing-plots
+[sharing]: /doc/user-guide/experiment-management/persisting-experiments
 
-`dvc exp save` creates <abbr>experiments</abbr> just like
-[`dvc exp run`](/doc/command-reference/exp/run), but does not require
-reproduction or setup of [pipeline stages](/doc/start/data-pipelines), making it
-possible to quickly start tracking, [comparing] and [persisting] experiments.
+<admon type="tip">
 
-[comparing]: /doc/user-guide/experiment-management/comparing-experiments
-[persisting]: /doc/user-guide/experiment-management/persisting-experiments
+Only files tracked by either Git or DVC are saved to the experiment unless you
+use `--include-untracked` (`-I`) on untracked files explicitly (see an
+[example](#examples) below).
+
+</admon>
+
+[Review] your experiments with `dvc exp show`. Successful ones can be [made
+persistent] by restoring them via `dvc exp branch` or `dvc exp apply` and
+committing them to the Git repo. Unnecessary ones can be [cleared] with
+`dvc exp gc`.
+
+[review]: /doc/user-guide/experiment-management/comparing-experiments
+[made persistent]: /doc/user-guide/experiment-management/persisting-experiments
+[cleared]: /doc/user-guide/experiment-management/cleaning-experiments
 
 ## Options
 
 - `-n <name>`, `--name <name>` - specify a [unique name] for this experiment. A
   default one will be generated otherwise, such as `exp-f80g4` (based on the
   experiment's hash).
 
-- `-I <path>`, `--include-untracked <path>` - specify untracked file(s) to be
-  included in the saved experiment. Multiple files can be specified.
+- `-I <path>`, `--include-untracked <path>` - specify an untracked file or
+  directory to be included in the experiment. This option can be used multiple
+  times.
 
-- `-f`, `--force` - overwrite the experiment if an experiment with the same name
-  already exists.
+- `-f`, `--force` - save the experiment even if no tracked files change (similar
+  to `dvc commit -f`).
```

Comment:
> This matches what the CLI `-h` output says but I'm not sure which one is the actual behavior rn.

Reply:
The help text is outdated and being updated to match the docs.

Reply:
K, but so what's the behavior like? `repro -f` or `commit -f`? Thanks

Reply:
https://github.com/iterative/dvc/pull/8729

Reply:
> save also errors out if nothing changed in the workspace, --force overrides this

Should include that info ⏳

Reply:
> the dvc commit part of exp save is always forced

Is that also the case for `exp run` then? I wonder wether we should mention this in both refs 🤔 

For now I updated the `exp save -f` description to reflect that info. @dtrifiro (thanks BTW). PTAL

Reply:
For `exp run` it is a bit different (and confusing): `--force` applies to both to the experiment name (overwrites the experiment if it already exists) and stage execution (same meaning as `repro -f`)

Reply:
Planning on getting rid of this behaviour in https://github.com/iterative/dvc/pull/8743. With that `-f` only applies to the experiment name

Reply:
> For `exp run` it is a bit different (and confusing): `--force` applies to both to the experiment name (overwrites the experiment if it already exists) and stage execution (same meaning as `repro -f`)

@dtrifiro I think this part is still true? `-f` will still force the stage to be executed again. Alternatively, we could make overwriting the experiment name a separate flag like `--overwrite/-o`.

Reply:
Lmk if this needs an update. 👍🏼